File: tests/Generator/resource/visualbasic/complex/Response.vb
Imports System.Text.Json.Serialization

' Describes the response of the operation
Public Class Response
    <JsonPropertyName("code")>
    Public Property Code As Nullable(Integer)

    <JsonPropertyName("contentType")>
    Public Property ContentType As Nullable(String)

    <JsonPropertyName("schema")>
    Public Property Schema As Nullable(PropertyType)

End Class

File: tests/Generator/resource/rust/complex/response.rs
use serde::{Serialize, Deserialize};
use property_type::PropertyType;

// Describes the response of the operation
#[derive(Serialize, Deserialize)]
pub struct Response {
    #[serde(rename = "code")]
    code: Option<u64>,

    #[serde(rename = "contentType")]
    content_type: Option<String>,

    #[serde(rename = "schema")]
    schema: Option<PropertyType>,

}

